gojira.sh

```diff
@@ -819,12 +822,12 @@ main() {
     p_compose down -v
     ;;
   shell)
-    p_compose exec --index $GOJIRA_CLUSTER_INDEX $GOJIRA_TARGET gosh -l -i
+    run_command $GOJIRA_TARGET $GOJIRA_CLUSTER_INDEX gosh -l -i
     ;;
   shell@*)
     # remove shell@, anchored at the start
     local where=${ACTION/#shell@/}
-    p_compose exec --index $GOJIRA_CLUSTER_INDEX $where sh -l -i
+    run_command $where $GOJIRA_CLUSTER_INDEX sh -l -i
```

Comment:
`run_command` does not accept arbitrary arguments as it is, so these are running just `gosh` and `sh` (you can see it by running `gojira` with the `-V` flag).

Either quote them so they are just one argument, or we need to make changes to the `run_command` function (remove the default for the second argument, shift the first two arguments and use `$@` for args).

Reply:
Great point @eskerda. I opted to quote the "sh -l -i" and then handled [`SC2086`](https://github.com/koalaman/shellcheck/wiki/SC2086) shell check by adding quotes around the `$where` and `$GOJIRA_CLUSTER_INDEX` args as well.